docs/agents/sematext-agent/autodisco/resource-limits.md

```diff
@@ -0,0 +1,17 @@
+title: App Agent Resource Limits
+
+description:  Setting resource limits for App Agent pods.
+
+Sematext Agent spins up a new pod for every App Agent responsible for monitoring a particular service. The App Agent pod is configured to run with the following [request/limit](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container) specifications:
```

Comment:
Sematext Agent spins up a new pod for every App Agent responsible for monitoring a particular service

=>

Sematext Agent spins up a new Pod for every App Agent responsible for monitoring a particular service

Reply:
The App Agent pod is configured to run with the following [request/limit](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container) specifications:

=>

The App Agent Pod is configured to run with the following [request/limit](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container) specifications: